rplugin/python3/denite/ui/default.py

```diff
@@ -141,7 +141,7 @@ def init_buffer(self):
             # Create new buffer
             self._vim.call(
                 'denite#util#execute_path',
-                'silent ' + self._context['direction'] + ' new', '[denite]')
+                'silent keepalt ' + self._context['direction'] + ' new', '[denite]')
```

Comment:
@lambdalisue I've fixed that,  I counldn't open the travis log page due to bad network.